devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy

```diff
@@ -102,13 +103,21 @@ def parse(String description) {
 				} else {
 					log.warn "IAS ZONE REPORTING CONFIG FAILED- error code: ${descMap.data[0]}"
 				}
+			} else if (descMap?.clusterInt == zigbee.IAS_ZONE_CLUSTER && descMap.attrInt == 0x0001 && isBoschRadionMultiSensor()) {
```

Comment:
Does this work when the device is running locally?

Reply:
Double-checked, works for both configurations -> 
runLocally set to TRUE,  executeCommandsLocally set to FALSE
runLocally set to FALSE, executeCommandsLocally set to FALSE